easybuild/easyblocks/g/gcc.py

```diff
@@ -50,18 +50,33 @@
 from easybuild.tools.filetools import which, write_file
 from easybuild.tools.modules import get_software_root
 from easybuild.tools.run import run_cmd
+from easybuild.tools.systemtools import AARCH32, AARCH64, POWER, X86_64
 from easybuild.tools.systemtools import check_os_dependency, get_os_name, get_os_type
-from easybuild.tools.systemtools import get_gcc_version, get_shared_lib_ext
+from easybuild.tools.systemtools import get_cpu_architecture, get_gcc_version, get_shared_lib_ext
 from easybuild.tools.toolchain.compiler import OPTARCH_GENERIC
 from easybuild.tools.utilities import nub
 
 
+# Offloading stages to build
+NVPTX_TOOLS = 'NVIDIA_NVPTX_TOOLS'
+NVIDIA_NEWLIB = 'NVIDIA_NEWLIB'
+AMD_LLVM = 'AMD_GCN_LLVM'
+AMD_NEWLIB = 'AMD_GCN_NEWLIB'
+HOST_COMPILER = 'HOST_COMPILER'
+# Additional command symlinks to create
 COMP_CMD_SYMLINKS = {
     'cc': 'gcc',
     'c++': 'g++',
     'f77': 'gfortran',
     'f95': 'gfortran',
 }
+# Mapping of EasyBuild CPU architecture names to list of LLVM target architectures
+LLVM_ARCH_MAP = {
```

Comment:
Can't we import this from the Clang easyblock, where we already have this mapping?

```python
from easybuild.easyblocks.clang import DEFAULT_TARGETS_MAP as LLVM_ARCH_MAP
```

(note that the values there are *lists*, not sure why...)